pages/osx/coreaudiod.md

```diff
@@ -0,0 +1,10 @@
+# coreaudiod
+
+> Service for Core Audio on Mac OS.
+> Involved with audio input and output.
```

Comment:
We don't use more than 3 command description lines
```suggestion
> Service for Core Audio on macOS. Involved with audio input and output.
```

Reply:
```suggestion
> Service for Core Audio, Apple's audio system.
```